### PR TITLE
49 fixed recipe list on mobile

### PR DESCRIPTION
**Pull request by eyesurewould**
Fix for #49

### CODE REVIEW

**Comment by buildabeer-admin** [comment]
image
The Name now look really small on large screens.

On small screens (at least using desktop browser resizing) still has some trouble as well. 
image

**Comment by eyesurewould** [comment]
Sorry about that. I tried adding a breakpoint using an @media query and
didn't add any for larger screens. Clearly that won't fit the approach
you've used for breakpoints.

Unfortunately, I haven't seen where you've defined your breakpoints yet, so
maybe you can point me to that code or explain the responsive strategy.

**Comment by buildabeer-admin** [comment]
Ooh, I see what you mean. The project is using Twitter bootstrap for CSS,
which gives xs, sm, md, lg, and xl screen size definitions.

**Comment by eyesurewould** [comment]
Oh. Duh!

OK, clearly you can reject that PR and I'll give that another go.

**Comment by eyesurewould** [comment]
I see you've got Bootstrap 4.2.1 (per the package.json) but I don't see a single media breakpoint. For example, I expected to see something like this:
`@include media-breakpoint-up(xs)`

In addition, I don't see a node_modules/bootstrap/scss folder (but that may not be an issues).

I'm admittedly new to Sass, so if I'm missing something, just let me know.

**Comment by buildabeer-admin** [comment]
I don't remember what I did for the scss, it does look like it should be node_modules/bootstrapp/scss. If the components are styling it's gotta be getting pulled from somewhere, right? Haha...

Is this the kind of thing you are looking for:
https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

**Comment by eyesurewould** [comment]
Yeah, that's exactly what I would expect to work but that folder is missing
on my repo. The bootstrap package looks to be the same on the main repo. Do
you have node_modules/bootstrapp/scss on your local repo?

Things render in the browser using the supplied scss which tells me the
compilation steps are working correctly, so nothing is really wrong. I just
need to figure out where to supply styling for the "sm" breakpoint (eg.
under 767px).

**Comment by buildabeer-admin** [comment]
Apologies if I'm misinterpreting anything. I won't be able to check if my
installed project has it until the morning. But you can put new styling w/
the breakpoint in styles.scss for global styling, or a components specific
scss file to apply it exclusive.

**Comment by buildabeer-admin** [comment]
I do have bootstrap/scss in my node modules. Which should be getting imported in src/styles.scss on the line '@import "~bootstrap/scss/bootstrap";. I am about 90% confident though that if you do something along the lines of this:

@include media-breakpoint-up(sm) {
  .some-class {
    //styles here
  }
}

Then it will apply that styling only on the sm screen size, kind of deal. Putting it in styles.scss will make it global.

File: brew_wizard_ui/src/app/recipes/recipe-list/recipe-list.component.scss [hunk 1]
```diff
@@ -84,4 +94,58 @@
...
+  .status {
+    padding-left: 30%;
+  }
+
+  .mini-row {
+    h5 {
+      font-size: smaller;
+      margin-bottom: -5px;
+    }
+    height: 20px;
```
Comment (buildabeer-admin) [inline brew_wizard_ui/src/app/recipes/recipe-list/recipe-list.component.scss]: I like the look if we drop the height on mini-row here, and change the recipe name from smaller to medium (or pick a number). Or small size to 70% w/ recipe name as smaller,  I think the recipe name needs a higher significance than the creator name. the smaller/70% I think looks alright on mobile, on desktop it looks too small.

**Comment by buildabeer-admin** [comment]
Are the fontawesome icons showing up for you by the way? My icons aren't showing at the moment for some reason.

**Comment by buildabeer-admin** [comment]
I think I just realized what's going on! When you create a PR, are you doing it from an isolated branch? It looks like your changes are stacking on top of eachother, so when you do a new PR it has the previous changes and new changes. I'm going to merge everything through, but I have some minor modifications on the changes as well. It looks like a linter was ran over the code (which is good, made reviewing a pain), and a lot of fantastic looking UI/UX changes that I am excited about! It does look like Save to PDF was broken in the process, and I'll create a bug for that.

I did the merge off of pull #62, which has 6 commits compared to this ones 4.

Closes #49 
Closes #59

**Pull request closed without merging**